css/pages/_content.scss

```diff
@@ -46,6 +67,59 @@
         font-size: 16px;
         line-height: 1.5;
         font-weight: 500;
+
+        @include media("<tablet") {
+            margin: 0 0 8px;
+            display: block;
+            border: 1px solid $grey-3;
+        }
+
+        &[width-50],
+        &.width-50 {
+            max-width: 50%;
+            width: 50%;
+
+            @include media("<laptop") {
+                max-width: 100%;
+                width: 100%;
+            }
+
+            td,
+            th {
+                word-break: break-word;
+            }
+        }
+
+        &[width-100],
+        &.width-100 {
+            max-width: 100%;
+            width: 100%;
```

Comment:
Why do we duplicate widths if we have them by default?

Reply:
it's specific classes for content manager.
table can have another default width due to js